starling/display/Sprite.hx

```diff
@@ -112,6 +112,7 @@ class Sprite extends DisplayObjectContainer
      */
     public function flatten(ignoreChildOrder:Bool=false):Void
     {
+        return;
```

Comment:
should we now just remove everything related to flattening?

Reply:
Yeah, I was also thinking is that something we should do on this ticket or have a separate clean up ticket. To me it sounds better to have a separate ticket for clean up. The rationale behind it is that it is that we have less changes on this branch and it is gonna be easier when after some years you'd like to check the git history to see what has been required to implement autobatching in starling. 
Don't mind to do the cleanup on this branch though... :)